alacritty/src/cli.rs

```diff
@@ -161,16 +161,16 @@ fn option_as_value(option: &str) -> Result<Value, serde_yaml::Error> {
 fn parse_class(input: &str) -> Result<Class, String> {
     match input.find(',') {
         Some(position) => {
-            let general = input[position + 1..].to_owned();
+            let instance = input[position + 1..].to_owned();
```

Comment:
Not sure why we haven't done this, but seems like this would be much cleaner with `split_once`, right?

Reply:
It was too new when this was written.